foundry.toml

```diff
@@ -11,7 +11,7 @@ sort_imports = true
 [profile.default]
 solc_version = '0.8.25'
 libs = ["node_modules", "lib"]
-optimizer_runs = 500
+optimizer_runs = 50000
```

Comment:
ahh, makes sense... i don't think project use the optimizer runs so high, there's a threshold where it stops being "gas efficient" and it starts being very expensive to deploy, in order to make the PR understandable, i'd remove this change and discuss this on a more _ceteris paribus_ approach..